pylint/config/_pylint_config/__init__.py

```diff
@@ -7,5 +7,7 @@
 Everything in this module is private.
 """
 
-from pylint.config._pylint_config.main import _handle_pylint_config_commands  # noqa
-from pylint.config._pylint_config.setup import _register_generate_config_options  # noqa
+from pylint.config._pylint_config.main import _handle_pylint_config_commands
+from pylint.config._pylint_config.setup import _register_generate_config_options
+
+__all__ = ("_handle_pylint_config_commands", "_register_generate_config_options")
```

Comment:
Weren't you against adding private api in ``__all__`` previously ? I think it's ok myself, it's also a tool to make our live easier and imports clearer even the private one. But we might want to do that elsewhere too in that case.

Reply:
Yeah I still am. But `mypy` complained about this and we have a docstring + module name that indicate that this is private. If anybody still uses it is their fault.